src/wiki/development/development-builds.md

```diff
@@ -15,7 +15,8 @@ Non-portable binaries are currently available for Windows as a system install, m
 
 Portable dev-builds are also provided; for both Windows and Linux.
 
-For Debian and Arch, you can use these packages for the latest development versions:  
-[![polymc-git](https://img.shields.io/badge/aur-polymc--git-blue)](https://aur.archlinux.org/packages/polymc-git/)
-[![polymc-git](https://img.shields.io/badge/mpr-polymc--git-orange)](https://mpr.makedeb.org/packages/polymc-git)  
+For Arch, Debian and Gentoo, respectively, you can use these packages for the latest development versions:
+
+[![polymc-git](https://img.shields.io/badge/aur-polymc--git-blue)](https://aur.archlinux.org/packages/polymc-git/)   [![polymc-git](https://img.shields.io/badge/mpr-polymc--git-orange)](https://mpr.makedeb.org/packages/polymc-git)   [![polymc-9999](https://img.shields.io/badge/gentoo-polymc--9999-purple)](https://packages.gentoo.org/packages/games-action/polymc)
+
 For Flatpak builds, you can use [flathub-beta](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111).
```

Comment:
Maybe it's time to remove flathub beta from here, it didn't actually get updated with devbuilds since 1.1.0 released

Reply:
hmm that probably isn't a bad idea.